src/ray/raylet/task.cc

```diff
@@ -46,6 +46,16 @@ void Task::CopyTaskExecutionSpec(const Task &task) {
   ComputeDependencies();
 }
 
+std::string TaskToFlatbuffer(const std::vector<ObjectID> *dependencies,
+                             const TaskSpecification *task_spec) {
```

Comment:
Why not make this a method of `Task`?

Reply:
Also, maybe `SerializeAsString` or `SerializeTaskAsString`? It might seem like this method converts the task to a flatbuffer object.

Reply:
I renamed it to SerializeTaskAsString; we cannot make this a method of Task because we cannot instantiate a Task in Cython on the stack at the moment (because it has a non-trivial constructor). We could make it static, but I think the current solution is good too.